src/mongoose_shapers.erl

```diff
@@ -0,0 +1,54 @@
+-module(mongoose_shapers).
```

Comment:
why not mongoose_shaper?
Single form is more popular

Reply:
ok, all the same to me :)